components/scream/src/physics/shoc/shoc_ic_cases.cpp

```diff
@@ -0,0 +1,185 @@
+#include "shoc_ic_cases.hpp"
+#include "shoc_constants.hpp"
+#include "share/util/scream_utils.hpp"
+#include "share/scream_assert.hpp"
+
+namespace scream {
+namespace shoc {
+namespace ic {
+
+namespace {
+
+// Reference elevations for data interpolation.
```

Comment:
Ideally we'd have multiple cases, so it might be good to encourage that by bundling all of this is a class, `StandardCase` or something like this.